man/systemd-vmspawn.xml

```diff
@@ -142,38 +142,54 @@
           </listitem>
         </varlistentry>
 
-      <varlistentry>
-        <term><option>--qemu-gui</option></term>
+        <varlistentry>
+          <term><option>--initrd=</option><replaceable>PATH</replaceable></term>
```

Comment:
Are we sure that qemu merges multiple initrds if more than one is specified?

Reply:
I'm not super sure, it doesn't seem to break.
It was proposed here: https://github.com/systemd/systemd/pull/29960#discussion_r1425688080

Reply:
To reiterate here, it does not merge initrds, we must do this ourself if we want this feature. (I think this should be considered in the scope of whatever is building the image not necessarily in vmspawn's scope.)